{"level":"error","message":"could not get the notes could not get the notes","stack":"Error: could not get the notes\n    at C:\\Notes\\Routes\\note.js:14:11\n    at newFn (C:\\Notes\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Notes\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Notes\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Notes\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at newFn (C:\\Notes\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Notes\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Notes\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Notes\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Notes\\node_modules\\express\\lib\\router\\index.js:275:10)"}
{"level":"error","message":"Something Failed during startup Something Failed during startup","stack":"Error: Something Failed during startup\n    at Object.<anonymous> (C:\\Notes\\app.js:25:7)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"Something failed miserably Something failed miserably","stack":"Error: Something failed miserably\n    at Object.<anonymous> (C:\\Notes\\app.js:30:26)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"could not get the notes could not get the notes","stack":"Error: could not get the notes\n    at C:\\Notes\\Routes\\note.js:14:11\n    at newFn (C:\\Notes\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Notes\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Notes\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Notes\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at newFn (C:\\Notes\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Notes\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Notes\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Notes\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Notes\\node_modules\\express\\lib\\router\\index.js:275:10)"}
{"level":"info","message":"info"}
{"level":"info","message":"server is running on port 3000....."}
{"level":"info","message":"server is running on port ....."}
{"level":"info","message":"info"}
{"level":"info","message":"info"}
{"level":"info","message":"server is running on port 3000....."}
{"level":"info","message":"server is running on port 3000....."}
